src/components/login-form.tsx

```
@@ -0,0 +1,115 @@
+              width={40}
+              height={40}
+            />
+          </Button>
+        </Form>
+
+        <div className="relative">
+          <div className="absolute inset-0 flex items-center">
+            <span className="w-full border-t" />
+          </div>
+          <div className="relative mx-auto flex w-[100%] items-center">
+            <div className="flex-1 border-2 border-lime-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)